popx/migrator_test.go

```diff
@@ -4,144 +4,28 @@
 package popx_test
 
 import (
-	"bytes"
 	"context"
 	"embed"
-	"fmt"
-	"os"
-	"strings"
 	"testing"
 
 	"github.com/gobuffalo/pop/v6"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/ory/x/dbal"
 	"github.com/ory/x/logrusx"
-	"github.com/ory/x/pkgerx"
 	. "github.com/ory/x/popx"
-	"github.com/ory/x/sqlcon/dockertest"
 )
 
 //go:embed stub/migrations/transactional/*.sql
 var transactionalMigrations embed.FS
 
-func TestMigratorUpgrading(t *testing.T) {
```

Comment:
This test was mostly skipped anyway, and I removed the "legacy" migrator with this PR, so no need to test compatibility anymore.